quinn-proto/src/congestion/bbr/bw_estimation.rs

```diff
@@ -43,10 +43,12 @@ impl BandwidthEstimation {
             None => return,
         };
 
-        let send_rate = if self.sent_time > prev_sent_time {
+        let send_rate = if self.sent_time > Some(prev_sent_time) {
+            // self.sent_time can't be None if it's greater than a Some value
+            let sent_time = self.sent_time.unwrap();
```

Comment:
I would also prefer something more natural than this combination of unwrapping and being clever about `Option<T>: Ord` in a way that requires a comment to explain, e.g. would `let send_rate = self.sent_time.filter(|sent_time| sent_time > prev_sent_time).unwrap_or(u64::MAX)` work?.

Edit: I see you added another commit, thanks! Just to be clear, it would be good if you'd squash it into the relevant commit to form a clean history. Maybe you know that, but GitHub doesn't provide me a great way to tell when you're done making changes in this situation.

Feel free to go with either the `match` based approach or the `filter` based approach. I think condensing it to a one-liner sounds neat but it's a minor difference.